### ISSUE TITLE
Create guide to self-hosting research object data on IPFS

### ISSUE THREAD

**Issue by m0ar** [desci-labs/nodes#547]
Create instructions for self-hosting the data in a research object.

Target audience: entities using Nodes to publish and discover, but not use DeSci Labs for data hosting.

Content draft:
- Overview of the Nodes / codex / IPFS / dPID building blocks
- IPFS basics
    - Public / private caveats
- Infrastructure requirements
- Kubo setup
- Adding data to IPFS
    - Default path: add local filetree as unixFS
    - Notes on formats
- Adding the resulting CIDs as external data in Node

**Comment by m0ar** [comment]
Looks OK @pkoellinger ?

**Comment by pkoellinger** [comment]
Yes, this looks great! Let's do it! Do you want to set this up in a Gitbook we can link to, or do you have something else in mind?

**Comment by pkoellinger** [comment]
Title idea: Guide to self-hosting research objects with CODEX and the DeSci Publish interface

**Comment by m0ar** [comment]
> Yes, this looks great! Let's do it! Do you want to set this up in a Gitbook we can link to, or do you have something else in mind?

Not sure! Maybe a dedicated gitbook for integration makes sense. Or a section somewhere in the existing user docs, don't have a good grasp on those tbh.
